src/bun.js/bindings/JSValue.zig

```diff
@@ -1559,10 +1569,26 @@ pub const JSValue = enum(i64) {
         };
     }
 
+    /// Runtime conversion to an object. This can have side effects.
+    ///
+    /// For values that are already objects, this is effectively a reinterpret
+    /// cast.
     pub fn toObject(this: JSValue, globalThis: *JSGlobalObject) *JSObject {
         return cppFn("toObject", .{ this, globalThis });
     }
 
+    /// Statically cast a value to a JSObject.
+    ///
+    /// Returns _null_ for non-objects. Use `toObject` to runtime-cast them instead.
+    pub fn asObject(this: JSValue) ?*JSObject {
```

Comment:
This is a bit backwards. `getObject` should be optional, and `asObject` should assert. That's how JSC does it in their methods.

Reply:
I agree it's backwards, but that's how `JSC` has named it.